perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
Yes, you guessed it right. You need to add some tests for this function.
BTW, for this function, why don't you use this with [map()](https://docs.python.org/3/library/functions.html#map)?